packages/react-notion-x/src/block.tsx

```diff
@@ -815,6 +815,23 @@ export const Block: React.FC<BlockProps> = (props) => {
     case 'transclusion_reference':
       return <SyncPointerBlock block={block} level={level + 1} {...props} />
 
+    case 'alias':
+      let blockPointerId = block?.format?.alias_pointer?.id
```

Comment:
try to use `const` if a variable doesn't change; you may need to surround the case block in braces (this is a small nit; just browsing 😄 )

Reply:
I'll update this! Thanks for taking a look!